specs/data-grid/data-grid.spec.md

```diff
@@ -0,0 +1,302 @@
+# Data Grid
+
+## Overview
+
+The `data grid` component enables authors to display an array of data in a tabular layout.  The layout can include a "header" region where a title can be displayed for each column.  
+
+![](./images/grid.png)
+
+`Data grid` actually consists of a number of components that work together:
+- `data grid`:  The top level container element
+- `data grid header`: Displays column titles
+- `data grid header cell`: Displays the title for a single column of cells
+- `data grid row`: Displays a single row of data associated with a single record
+- `data grid cell`: Displays a single cell of data within a row
+
+### Use Cases
+Any time an author wants to display tabular data.
+
+### Non-goals
+- The initial version of the component will not support virtualization or pagination to display large data sets but should be architected to support both in the future.
+  
+### Features
+- Generates a data grid layout based on provided data.
+- Authors can take advantage of multiple customization points to control the grid display
+- manages keyboard navigation across the grid
+
+### Risks and Challenges
+- not yet clear how cells with focusable children work 
+- we're not requiring unique identifiers per row/data item, do we need to to ensure stable relationships between data rows and component representations?
```

Comment:
```suggestion
- It is not yet clear how cells with focusable children work.
- Unique identifiers are not required per row/data item, this may cause an unstable relationships between data rows and component representations.
```